pages/linux/xsel.md

```diff
@@ -0,0 +1,16 @@
+# xsel
+
+> Selection manipulation tool.
+> For example xsel can copy STDIN to clipboard or print clipboard to STDOUT.
```

Comment:
I'd write the second sentence as:

"For example, xsel can copy STDIN to the clipboard, or print clipboard contents to STDOUT."